### PR TITLE
lestarch: release notes for 2.0

### PR DESCRIPTION
**Pull request by LeStarch**
| | |
|:---|:---|
|**_Originating Project/Creator_**| |
|**_Affected Component_**|  |
|**_Affected Architectures(s)_**|  |
|**_Related Issue(s)_**|  |
|**_Has Unit Tests (y/n)_**|  |
|**_Builds Without Errors (y/n)_**|  |
|**_Unit Tests Pass (y/n)_**|  |
|**_Documentation Included (y/n)_**|  |

---
## Change Description

2.0 is neigh....

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,22 +94,36 @@ The typed port connections provide strong compile-time guarantees of correctness
...
 
-#### Release 1.01
+* F´ tooling (fprime-util and fprime-gds) should be installed using `pip install fprime-tools fprime-gds`
+* `Os::File::open` with the mode CREATE will now properly respect O_EXCL and error if the file exists. Pass in `false` as the final argument to override. 
+* Revise uses of `Fw::Buffer` to correct usage of member functions using camel case.  E.g. `Fw::Buffer::getsize` is now `Fw::Buffer::getSize`
+* The ground interface chain has been refactored. Projects may switch to using `Svc::Framer`, `Svc::Deframer`, and any implementor of `Drv::ByteStreamDriverModel` to supply the data.  See: TBD.  To continue using the old interface with the GDS run `fprime-gds --comm-checksum-type fixed`.
+* `Svc::FileDownlink` may now be configured to turn off certain errors
+* `Svc::BufferManager` has been reworked to remove errors. When instantiating it please supply a memory allocator. `Ref` has an example of this.
+* A project must now enable F´ framework UTs using `fprime-util generate --ut -DFPRIME_ENABLE_FRAMEWORK_UTS=ON`.  Project defined UTs are unaffected.
+* Dictionaries, binaries, and other build output now are written to a deployments `build_artifacts` folder.
```
Comment (Joshua-Anderson) [inline README.md:109]: Do we want to add a note about the removal of the make based build system?

Do we want to add notes about features we added to this release?

Comment (LeStarch) [reply README.md:109]: Are there features missing?  I thought I added most of the "big" features.  I identified them by crawling all the PRs since 1.5.4.

Comment (Joshua-Anderson) [reply README.md:109]: A minor missed issue is adding a filedownlink port and queue. But the notes discuss the modifications in terms of needed migrations, but I feel that kinda downplays the improvements made. We should advertise and be proud of the new ground chain and re-written IP drivers!

Comment (LeStarch) [reply README.md]: Ok I addressed this concerns

**Review by Joshua-Anderson — APPROVED** [review]
:rocket:

**Pull request merged**